spec/features/basic_auth_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "Basic auth" do
+  around do |example|
+    ENV["BASIC_AUTH_PASSWORD"] = "password"
```

Comment:
question - do we need to specify that we have a our username in these scenarios?